stacks/ecs.yml

```diff
@@ -33,11 +33,7 @@ Parameters:
     Type: String
   OpsErrorMessagesSnsTopicArn:
     Type: String
-  SecretsBase:
-    Type: String
-  SecretsKey:
-    Type: String
-  SecretsRegion:
+  InfrastructureStorageStackName:
```

Comment:
I figure better to pass in this one param, already in use in the root stack, than add in any secret specific params.